helm/linkerd2-multicluster-app/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "v1"
+# this version will be updated by the CI before publishing the Helm tarball
+appVersion: edge-XX.X.X
+description: |
+  The Linkerd-Multicluster extension contains resources to support multicluster
+  linking to remote clusters
+home: https://linkerd.io
+keywords:
+- service-mesh
+kubeVersion: ">=1.20.0-0"
+name: "linkerd-multicluster"
```

Comment:
name should match the repository name without `-app` (See https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/#11-create-and-configure-a-repo-for-the-app)